docs/resources/ipam_ip.md

```diff
@@ -0,0 +1,113 @@
+---
+subcategory: "IPAM"
+page_title: "Scaleway: scaleway_ipam_ip"
+---
+
+# scaleway_ipam_ip
+
+Books and manages Scaleway IPAM IPs.
+
+## Example
+
+### Basic
+
+```hcl
+resource "scaleway_vpc" "vpc01" {
+  name = "my vpc"
+}
+
+resource "scaleway_vpc_private_network" "pn01" {
+  vpc_id = scaleway_vpc.vpc01.id
+  ipv4_subnet {
+    subnet = "172.16.32.0/22"
+  }
+}
+
+resource "scaleway_ipam_ip" "ip01" {
+  source {
+    private_network_id = scaleway_vpc_private_network.pn01.id
+  }
+}
+```
+
+### Request a specific IPv4
+
+```hcl
+resource "scaleway_vpc" "vpc01" {
+  name = "my vpc"
+}
+
+resource "scaleway_vpc_private_network" "pn01" {
+  vpc_id = scaleway_vpc.vpc01.id
+  ipv4_subnet {
+    subnet = "172.16.32.0/22"
+  }
+}
+
+resource "scaleway_ipam_ip" "ip01" {
+  address = "172.16.32.7/22"
```

Comment:
Is it a single IP /32 or a range?

Reply:
It's a single IP in CIDR notation that falls in the range of the PN's subnet `172.16.32.0/22`